modules/olm-fb-catalogs-structure.adoc

```diff
@@ -35,6 +35,7 @@ catalog
 │       └── packageB.v0.1.0.clusterserviceversion.yaml
 └── packageC
     └── index.json
+    └── deprecations.yaml
 ----
 
-This recommended structure has the property that each subdirectory in the directory hierarchy is a self-contained catalog, which makes catalog composition, discovery, and navigation trivial file system operations. The catalog could also be included in a parent catalog by copying it into the parent catalog's root directory.
+This recommended structure has the property that each subdirectory in the directory hierarchy is a self-contained catalog, which makes catalog composition, discovery, and navigation trivial file system operations. The catalog could also be included in a parent catalog by copying it into the parent catalog's root directory.
```

Comment:
I know this is unmodified by the PR, but ... this whole "I contain multitudes" concept, IMHO, doesn't clarify anything.  In the vernacular, `catalog` refers to any conglomeration of information sufficient to install `a thing`.  
While it's true that a collection of FBC may be subdivided as makes sense (for e.g. per-package), catalog/sub-catalog is just confusing.

In pipelines, we refer to the per-package atoms as `FBC fragments` or `contributions`, which upon aggregation become `catalogs`.  But those descriptors are useless outside the pipeline context.  ☹️ 

Perhaps rewording similar to 
```
This recommended structure has the property that each subdirectory in the hierarchy is intuitively organized per package, which also makes catalog composition/discovery/navigation trivial file system operations.
```
I just don't know what to do with the last sentence at all. 😦 @joelanford ?

Reply:
Leaving unresolved for further discussion + a follow-up PR.